content/francineblanc/how-to-use-git-for-beginners.md

```diff
@@ -0,0 +1,219 @@
+
+Now, with the ```.gitignore file``` made and the ```.DS_Store``` file added, running ```git status``` in the terminal shows the following output:
+
+![git-status-untracked-with-gitignore](git-status-untracked-with-ignore.png)
+
+## Making Changes and Adding Files to the Staging Area
+In order for Git to start tracking changes, files need to be added to the Staging Area. This can be done with the command:
+```
+git add <filename>
+```
+where filename is the name and extension of the file to be added to the staging area. Multiple files can also be added with the syntax:
+
+```
+git add <filename1> <filename2> <filename3>
+```
+
+If all files in the Working Directory need to be tracked, the syntax
+
+```
+git add .
+```
+can be used.
+
+Once a file has been added, ```git status``` can be run in the terminal/command prompt to check that the file is in the Staging Area. Git will show the changed files to be committed in <span style="color:green">green</span> text like so:
+
+![git-status-untracked-with-gitignore](git-add-and-status.png)
+
+You can find out more about the additional extensions to ```git add``` in the [Add] section of the Codecademy Docs for Git.
+
+### Oops! I didn't want to add that!
+There may be occasions where a file has been incorrectly added to the Staging Area. This can be undone with the command ```git reset <filename>```. This will not affect the changes done to the file in any way, it will simply remove it from the staging area. 
+
+In the case where all files needs to be unstaged, the command ```git reset``` can be run without adding further arguments.
+
+Note that as of ```git version 2.24``` there is alternative syntax to unstage files, which is ```git restore --staged <filename>``` where ```filename``` is the name of the file to be unstaged.
+
+## Committing Changes
+Committing is generally the last step of the Git workflow and is generally thought of being a snapshot of a project at a particular time. In this stage, changes in the Staging Area are saved inside the local repository, which is achieved with the command ```git commit -m``` followed by a space and a short message explaining the commit in quotes, for example:
+```
+git commit -m "Add title and description to intro file"
+```
+There are general conventions for writing a commit message including making sure they are no more than 50 characters long and writing them in the imperative or present tense. Messages should always be clear and confirm what the change made is so that others can easily see what has changed. 
```

Comment:
For what it's worth, there is a resource called [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) that might be a good fit to link here:

```suggestion
There are general [conventions](https://www.conventionalcommits.org/en/v1.0.0/) for writing a commit message including:

- Making sure they are no more than 50 characters long.
- Writing them in the imperative or present tense.

Messages should always be clear and informative to others who may go back to confirm what changes were made.
```